src/Parser/Core/Modules/Items/Legion/AntorusTheBurningThrone/KhazgorothsCourage.js

```diff
@@ -54,6 +80,36 @@ class KhazgorothsCourage extends Analyzer {
     }
   }
 
+  get highestSecondaryStat(){
+    const haste = this.statTracker.currentHasteRating;
+    const crit = this.statTracker.currentCritRating;
+    const mastery = this.statTracker.currentMasteryRating;
+    const versatility = this.statTracker.currentVersatilityRating;
+    if(haste >= crit && haste >= mastery && haste >= versatility){
+      return 'haste';
+    } else if(crit >= mastery && crit >= versatility){
+      return 'crit';
+    } else if(mastery >= versatility){
+      return 'mastery';
+    } else{
+      return 'versatility';
+    }
+  }
+
+  _makeStatChange(ratingChange, statName, eventReason) {
```

Comment:
You can make an object field programatically with [ ], so here you could skip the if/else by just doing `const statChangeObj = { [statName]: ratingChange };`

Reply:
Cheers, fixed now.